tools/accuracy_checker/accuracy_checker/preprocessor/geometric_transformations.py

```diff
@@ -29,7 +29,8 @@
 from ..dependency import ClassProvider
 from ..logging import warning
 from ..preprocessor import Preprocessor
-from ..utils import contains_all, get_size_from_config, string_to_tuple, get_size_3d_from_config
+from ..utils import contains_all, get_size_from_config, string_to_tuple, get_size_3d_from_config,
+                    get_stride_from_config
```

Comment:
it does not work, you need to wrap multiline import statement in the brackets for treat it as single line.
something like that:
```python
from ..utils import (
contains_all, get_size_from_config, string_to_tuple, get_size_3d_from_config, get_stride_from_config
)
```